README.md

```diff
@@ -8,152 +8,225 @@
 
 [Toolbox](https://github.com/containers/toolbox) is a tool that offers a
 familiar package based environment for developing and debugging software that
-runs fully unprivileged using [Podman](https://podman.io/).
+runs fully rootless using [Podman](https://podman.io/) and integrates with
+your system.
 
-The toolbox container is a fully *mutable* container; when you see
-`yum install ansible` for example, that's something you can do inside your
-toolbox container, without affecting the base operating system.
-
-This is particularly useful on
-[OSTree](https://ostree.readthedocs.io/en/latest/) based operating systems like
-[Fedora CoreOS](https://coreos.fedoraproject.org/) and
-[Silverblue](https://silverblue.fedoraproject.org/).  The intention of these
-systems is to discourage installation of software on the host, and instead
-install software as (or in) containers.
-
-However, this tool doesn't *require* using an OSTree based system — it
-works equally well if you're running e.g. existing Fedora Workstation or
-Server, and that's a useful way to incrementally adopt containerization.
-
-The toolbox environment is based on an [OCI](https://www.opencontainers.org/)
-image. On Fedora this is the `fedora-toolbox` image. This image is used to
-create a toolbox container that seamlessly integrates with the rest of the
-operating system.
-
-## Usage
+## Basic usage
 
 ### Create your toolbox container:
-```
+
+```shell
```

Comment:
Nice. I didn't know that it's possible to specify the *language* like that.

However, it seems that it would be [better](https://stackoverflow.com/questions/20303826/highlight-bash-shell-code-in-markdown-files) [to use](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) `console` instead of `shell`.

Reply:
And I didn't know there's `console` :). Let's use it!

Reply:
Submitted https://github.com/containers/toolbox/pull/598 for this part.